test/algorithms/simulation/sidb/operational_domain.cpp

```diff
@@ -281,6 +352,29 @@ TEST_CASE("BDL wire operational domain computation", "[operational-domain]")
             CHECK(op_domain_stats.num_evaluated_parameter_combinations == 50);
             CHECK(op_domain_stats.num_operational_parameter_combinations == 50);
             CHECK(op_domain_stats.num_non_operational_parameter_combinations == 0);
+
+            SECTION("3-dimensional")
+            {
+                const auto z_dimension = operational_domain_value_range{sweep_parameter::MU_MINUS, -0.35, -0.29, 0.01};
```

Comment:
@marcelwa what happens if I (as a user) use the wrong order of min, max? The OpDomain is empty, right? 
Should we add an assert, similiar to CLI?